app-admin/etcd-wrapper/files/etcd-wrapper.service.template

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=etcd{{SLOT}} (System Application Container)
+Conflicts=etcd{{SLOT}}.service
+
+[Service]
+Type=notify
+Environment=ETCD_NAME=%m
+Environment=ETCD_IMG_USER=etcd
+Environment="SOURCE_FILES={{SOURCE_FILES}}"
```

Comment:
How come this is indirect and sourced in the wrapper instead of just using two env files in the service?